guide/14-deep-learning/multi_object_tracking_using_object_tracker.ipynb

```diff
@@ -0,0 +1,383 @@
+{
```

Comment:
<p>Should this be changed to <code>tracker.update(frame)</code>?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/993/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>